modules/candidate_list/jsx/candidateListIndex.js

```diff
@@ -112,6 +122,13 @@ class CandidateListIndex extends Component {
           <a href={url}>{cell}</a></td>
       );
     }
+
+    if (column === 'Subproject') {
+      // If user has multiple subprojects, join array into string
+      let result = (cell) ? <td>{cell.join(',')}</td> : null;
```

Comment:
```suggestion
      let result = (cell) ? <td>{cell.join(', ')}</td> : null;
```

could we add spacing so it looks nicer? i'll approve after. testing it now